REQS.md

```
@@ -949,13 +953,15 @@ A self-service marketplace where members discover and book coaches directly, red
 
 ### 9.12 Guest & Drop-In Management
 
+<!-- BEGIN WIP: STORY-0036 -->
 **Guest Passes:**
 | Feature | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)